test_scripts/digest_cc.sh

```diff
@@ -0,0 +1,12 @@
+TMP_SEDFILE="build/embedded_config.sed"
```

Comment:
Please add a comment describing what this script does, and what input arguments are. It seems to me that the script is pretty Psiphon-specific, and, if so, we should mention that in the description comment.

What would this script do, if the script does not provide the required 2 arguments? Would be better to exit early on that.

Reply:
added description and noted that this is psiphon specific.